pkg/frinkiac/http/quote.go

```diff
@@ -6,27 +6,62 @@ import (
 	"fmt"
 	"net/http"
 	"net/url"
+	"strconv"
 	"strings"
 
-	"github.com/rs/zerolog/log"
 	"golang.org/x/net/html"
 )
 
-// QuoteResult represents a single result from a quote search
-type QuoteResult struct {
-	ImagePath string // e.g., /img/S09E22/202334/medium.jpg
-	Season    string // e.g., S09
-	Episode   string // e.g., E22
-	ID        string // e.g., 202334
-}
-
 // APISearchResult represents a single result from the Frinkiac API search endpoint
 type APISearchResult struct {
 	ID        int    `json:"Id"`
 	Episode   string `json:"Episode"` // e.g., S16E01
 	Timestamp int    `json:"Timestamp"`
 }
 
+// validateEpisodeFormat checks if the episode format is valid
+// Episode format is expected to be like "S16E01"
+func validateEpisodeFormat(episode string) error {
+	if len(episode) < 6 {
+		return fmt.Errorf("invalid episode format: %s", episode)
```

Comment:
Introduce a constant ErrInvalidEpisodeFormat and wrap it with a call that adds the value